packages/shared/src/broker.ts

```diff
@@ -77,9 +78,12 @@ const requestValidators = (network: ChainflipNetwork) => ({
           }),
         )
         .optional(),
+      z.number().optional(),
     ])
-    .transform(([a, b, c, d, e]) =>
-      [a, b, c, d, transformObjToSnakeCase(e)].filter(isNotNullish),
+    .transform(([a, b, c, d, e, f]) =>
+      [a, b, c, d, transformObjToSnakeCase(e), f].map((value) =>
+        isUndefined(value) ? null : value,
```

Comment:
Broker API accepts `null` as a parameter value in the case of optional params (both the `ccmMetadata` & `boost fee` are optional). 
It also doesn't throw when provided with more parameters then expected, so that means that this approach will work and is backwards compatible.